lib/src/elastic_client_impl.dart

```diff
@@ -265,18 +270,70 @@ class Client {
       results,
       suggestHits: suggestHits.isEmpty ? null : suggestHits,
       aggregations: aggResult.isEmpty ? null : aggResult,
+      scrollId: body['_scroll_id'] as String ?? null,
+    );
+  }
+
+  Future<SearchResult> scroll(String scrollId, Duration scroll) async {
+    final path = ['_search', 'scroll'];
+    final bodyMap = {'scroll_id': scrollId};
+    if (scroll != null) {
```

Comment:
Can you confirm that `scroll` can be null at this point? If not, we don't need this if.

Reply:
This check was unnecessary, so I deleted it.